usecase/usecase-wrt-linux-tests/samples/DisplayMode/README.md

```diff
@@ -0,0 +1,3 @@
+## Usecase Design
+
+This sample demonstrates the support for setting csp policies in manifest.json.
```

Comment:
s/the support for setting csp policies/display mode support/

Add a line for the spec: http://w3c.github.io/manifest/#display-member